apps/src/templates/projects/projectsModule.js

```diff
@@ -1,10 +1,16 @@
 /** @file Redux actions and reducer for the Projects Gallery */
 import {registerReducers} from '@cdo/apps/redux';
+import _ from 'lodash';
 
 // Action types
+
 const TOGGLE_GALLERY = 'projectsModule/TOGGLE_GALLERY';
+const ADD_OLDER_PROJECTS = 'projectsModule/ADD_OLDER_PROJECTS';
+const SET_PROJECT_LISTS = 'projectsModule/SET_PROJECT_LISTS';
+const SET_HAS_OLDER_PROJECTS = 'projectsModule/SET_HAS_OLDER_PROJECTS';
```

Comment:
Also each set of actions/reducer should be in its own file. The last line in this file `
registerReducers({selectedGallery, projectLists, hasOlderProjects});` probably belongs elsewhere (I'm not sure exactly where).